apps/web/src/services/iam/pages/admin/AdminUserGroupPage.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
+import { onUnmounted } from 'vue';
+
+import { ApiQueryHelper } from '@cloudforet/core-lib/space-connector/helper';
+import { PHorizontalLayout } from '@cloudforet/mirinae';
+
+import UserGroupAddModal from '@/services/iam/components/UserGroupAddModal.vue';
+import UserGroupAddUsersModal from '@/services/iam/components/UserGroupAddUsersModal.vue';
+import UserGroupHeader from '@/services/iam/components/UserGroupHeader.vue';
+import UserGroupStatusModal from '@/services/iam/components/UserGroupStatusModal.vue';
+import UserGroupTab from '@/services/iam/components/UserGroupTab.vue';
+import UserGroupTable from '@/services/iam/components/UserGroupTable.vue';
+import { useUserGroupPageStore } from '@/services/iam/store/user-group-page-store';
+
+const userGroupPageStore = useUserGroupPageStore();
+const userGroupPageState = userGroupPageStore.state;
+
+const userGroupListApiQuery = new ApiQueryHelper()
+    .setSort('name', true);
+
+const fetchUserGroup = async () => {
+    userGroupListApiQuery.setPageStart(userGroupPageState.pageStart).setPageLimit(userGroupPageState.pageLimit)
+        .setFilters(userGroupPageState.searchFilters);
+
+    await userGroupPageStore.listUsers({ query: userGroupListApiQuery.data });
+};
+
+const init = () => {
+    fetchUserGroup();
+};
+init();
```

Comment:
question: Is there a specific reason for not using a mount point or an immediately invoked function expression for the `init` part?

Reply:
Initially, it was written as a namely immediate execution function, but it was changed after receiving a review of the code.

**before code review**
I wanted to use the named immediate execution function because I wanted to convey meaning rather than the anonymous immediate execution function.
```
(function init() {
    fetchUserGroup();
})());
```
**after code review**

```
const init = () => {
    fetchUserGroup();
};
init();
```